main.tex

```diff
@@ -5,7 +5,7 @@
 \usepackage{amsthm}
 \usepackage{mathtext}
 \usepackage{mathtools}
-\usepackage[T1,T2A]{fontenc}
```

Comment:
I had to get rid of T1 to use LGR, otherwise pdflatex would complain on `Too many math alphabets used in version normal`. Not sure if it is harmless or not.